modules/apps/object/object-service/src/main/java/com/liferay/object/internal/entry/util/ObjectEntryUtil.java

```diff
@@ -76,6 +81,8 @@ public static JSONObject getPayloadJSONObject(
 			"classPK", baseModel.getPrimaryKeyObj()
 		).put(
 			"extendedProperties", extendedProperties
+		).put(
+			"originalExtendedProperties", originalExtendedProperties
 		).put(
 			"model" + modelClass.getSimpleName(), baseModel.getModelAttributes()
```

Comment:
@JhossephG sort alphabetically. 

`"originalExtendedProperties"` should be placed after `"originalDTO" + dtoConverterType`